cli/flox/src/commands/activate.rs

```diff
@@ -103,6 +104,25 @@ impl Activate {
             Err(e) => Err(e)?,
         };
 
+        self.activate(config, flox, concrete_environment, true, &[])
+            .await
+    }
+
+    /// This function contains the bulk of the implementation for
+    /// Activate::handle,
+    /// but it allows us to create an activation for use by `services start` or
+    /// `services-restart`.
+    ///
+    /// If self.start_services is true and services_to_start is empty, all
+    /// services will be started.
+    pub async fn activate(
+        self,
+        mut config: Config,
+        flox: Flox,
+        mut concrete_environment: ConcreteEnvironment,
+        start_watchdog: bool,
+        services_to_start: &[String],
```

Comment:
it's not quite clear wha motivated this PR, my best guess is the watchdog argument is in preparation for #1886, what about `start_services`?
Though i'm also a bit lost on the need for ephemeral activations with the decition to implement #1890

Reply:
Buffed up the PR description a bit, does that make more sense now?